tests/test_fips.py

```diff
@@ -239,12 +239,23 @@ def test_gcrypt_binary(container_per_test: ContainerData) -> None:
 
     """
 
-    container_per_test.connection.check_output(
-        "zypper --gpg-auto-import-keys -n ref && zypper -n in gcc libgcrypt-devel && zypper -n clean && "
+    c = container_per_test.connection
+
+    # Oneline command ready for copy-paste debugging purposes:
+    #   zypper --gpg-auto-import-keys -n ref && zypper -n in gcc libgcrypt-devel && zypper -n clean && gcc -Og -g3 fips-test-gcrypt.c -Wall -Wextra -Wpedantic -lgcrypt -o fips-test-gcrypt && mv fips-test-gcrypt /bin/fips-test-gcrypt
```

Comment:
I think we can drop this comment as we can infer this from the commands below.

Reply:
Ooops sorry, forget to delete